bootstrap.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import config
+from gateserver import db
+
+def db_create_tables():
+    print('Creating DB tables...', end='')
+    with open('./tables.sql', 'r') as f:
```

Comment:
Hmm, od suboru s koncovkou .sql by som cakal korektne sql. Tu su to sql fragmenty s `#` komentarmi (as opposed to `--`). Trochu zvlastne.
(Preco je to vlastne v sql subore a nie v python kode?)